presto-main/src/main/java/io/prestosql/execution/resourcegroups/InternalResourceGroup.java

```diff
@@ -166,7 +162,7 @@ public ResourceGroupInfo getFullInfo()
                     softConcurrencyLimit,
                     hardConcurrencyLimit,
                     maxQueuedQueries,
-                    DataSize.succinctBytes(cachedMemoryUsageBytes),
+                    DataSize.succinctBytes(cachedResourceUsage.getMemoryUsageBytes()),
```

Comment:
Now that you're unifying this, I would probably add cpu usage here for consistency.